website_sale_stock_available_display/readme/USAGE.rst

```diff
@@ -0,0 +1,8 @@
+#. Go to *'Website > Products > Products'*.
+#. Set *Show inventory on website and allow sales if not enough stock* in
+   inventory availability field.
+#. You can set a custom message that appears when available stock is less than
```

Comment:
Where?

Reply:
@Tardo it is in the eCommerce tag, after the "Availability field".